src/test/java/com/wikia/webdriver/common/core/configuration/ManualConfiguration.java

```diff
@@ -90,4 +90,9 @@ public String getAppiumIp() {
   public String getDeviceName() {
     return config.get("device-name");
   }
+
+  @Override
+  public String getDisableFlash() {
+    return config.get("disable-flash");
```

Comment:
if you put in config file:
```disable-flash: true```
this wont work - can't cast boolean to String.
A solution:
```String.valueOf(config.get("disable-flash"));```